lib/containers/engine.pm

```diff
@@ -125,6 +130,9 @@ if C<cmd> found then it will execute the given command into the container.
 The container is always removed after exit.
 if C<keep_container> is 1 the container is not removed after creation. Default to get removed
 when it exits or when the daemon exits
+if C<retry> is given, the command is being repeaded the given amount of times on failure
```

Comment:
typo on `repeaded`. However i am against the retry function. i cant stress this enough. from another point, also you introduce this but it is unrelated with the changes

Reply:
Will fix the typo. The discussion on `retry` is however outside the scope of this PR.